### DISCUSSION TITLE
Select input not working?

### DISCUSSION THREAD

**Discussion by xeno** [themesberg/flowbite-svelte#608] [Q&A]
Here's my site:

https://2bd.netlify.app/

If you scroll down to the Pricing block, my select input is not changeable. This is an astro site with a svelte component in an island. Basically, a svelte component embedded on the page. It doesnt matter if I remove the binding or not, it just sticks to the one value (and not the placeholder, I might add.

I've looked at the source for the select component, and I'm out of ideas. If I look at the events in console debugger, I can disable the change events and then the component works, so something is happening in flowbite maybe?

```svelte
<script>
    import { Select, Label } from 'flowbite-svelte';

    export let packages;

    let packageOption;

    const packageOptions = packages.map(p => ({
        name: p.fields?.title,
        value: p.fields?.id
    }));
</script>

<div class="py-48 my-24 bg-gradient-radial from-purple-900 to-slate-900 border border-purple-900">
    <div class="grid md:grid-cols-12 grid-col-1 items-center gap-[30px] mx-auto max-w-screen-lg">
        <div class="lg:col-span-5 md:col-span-6 order-2 md:order-1">
            <div class="shadow dark:shadow-gray-800 rounded-md p-6 bg-gray-50 dark:bg-slate-800 overflow-hidden">
                <img src="/logo/vector_badge_-_mono_white.svg" class="mx-auto w-16" alt="">
                <h5 class="text-center font-semibold text-2xl mt-3">Two Brothers Pricing</h5>
                <Label for="packageOption" class="sr-only">Choose A Package</Label>
                <Select id="packageOption" underline class="mt-2" bind:value={packageOption} placeholder="Choose A Package" items={packageOptions} />
            </div>
        </div>

        <div class="lg:col-span-7 md:col-span-6 order-1 md:order-2">
            <div class="lg:mr-10">
                <h4 class="my-4 md:text-3xl md:leading-normal text-2xl leading-normal font-semibold">Your Total: x</h4>

            </div>
        </div>
    </div><!--end grid-->
</div>
```

**Comment by shinokada** [comment]
I have tried using flowbite-svelte on Astro before and I found some interactive components didn't work.
Try to use [this setup](https://flowbite.com/docs/getting-started/astro/) with Astro.

**Reply by xeno** [reply]
If I disable the flowbite-svelte event listeners, it works. Either way, looks like I can't use it.

**Reply by shinokada** [reply]
Nope. It won't work.
